packages/rancher-alerting-drivers/charts/templates/_helpers.tpl

```diff
@@ -0,0 +1,89 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "rancher-alerting-drivers.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
```

Comment:
Will this always be the same length? (`trunc 63`)

Reply:
This template method is generated by the `helm create` command. The length of the name can vary but not longer than 63 charactors. 

I am not sure if I understand your question correctly :) 
Do I answer your question?

Reply:
Yes. That makes sense!****